docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3.4'
+services:
+  db:
+    environment:
+      - MYSQL_USER=foreman
+      - MYSQL_PASSWORD=foreman
+      - MYSQL_DATABASE=foreman
+      - MYSQL_DATADIR_ACTION=upgrade-auto
+    hostname: db.example.com
+    image: centos/mariadb-102-centos7
+    ports:
+      - 3306
+    restart: always
+    healthcheck:
+      test: ["CMD-SHELL", "nc -z 127.0.0.1 3306 || exit 1"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+    volumes:
+      - db:/var/lib/mysql/data
+
+  app: &app_base
+    image: quay.io/ohadlevy/foreman:containers
+    command: bundle exec bin/rails server
```

Comment:
I had to change this to `command: bundle exec bin/rails server -b 0.0.0.0` to access the app.